src/lib/containers/SynapseTable.tsx

```diff
@@ -113,6 +116,13 @@ export default class SynapseTable extends React.Component<QueryWrapperChildProps
     return facetAliases[facetName] || facetName
   }
 
+  public isAggregate(): boolean {
+    if (this.props.getLastQueryRequest && this.props.getLastQueryRequest().query) {
```

Comment:
this whole function could instead be - 
return AGGREGATE_REGEX.test(this.props.getLastQueryRequest!().query.sql)

Reply:
Wasn't sure if the last query request would always be set when this component was initially rendered, but I'll take your word for it!